nercst/core/coords_converter.py

```diff
@@ -0,0 +1,24 @@
+import astropy.units as u
+import xarray as xr
+
+from neclib import config
+from astropy.time import Time
+from neclib.coordinates import parse_frame
+from astropy.coordinates import SkyCoord
+from neclib.parameters import PointingError
+
+
+def add_celestial_coords(array: xr.DataArray, frame: str) -> xr.DataArray:
+
+    pe = PointingError("OMU1P85M")
```

Comment:
Is it OK that it supports only 1.85m?
If you assume that at this time, it looks good to me.